### DISCUSSION TITLE
download-only in monitor

### DISCUSSION THREAD

**Discussion by mani2jeff** [abraunegg/onedrive#2411] [Q&A] [answered]
I am intending to use onedrive to do regular backup of OneDrive to a local device. I have monitor running and it is doing the initial download. (and that looks great so far.) I would rather just have download-only in the monitor and all changes are in the cloud. No updates from the local device to the cloud. I haven't found any configuration options to make that happen. Did I miss something? Is that possible? I suspect it will work great as is, but the download-only option would provide some reassurance of the cloud version (instead of looking in the recycle bin).  Thx

**Comment by abraunegg** [comment, marked as answer]
@mani2jeff 
> Did I miss something?

Yes you did!

> Is that possible?

Yes it is - whilst the documents show the manual sync (`--synchronize`) you can use it with `--monitor` without issue.

It is potentially better to add the following to your config file, so that it is the 'default' method of action:
```
download_only = "true"
```

**Reply by mani2jeff** [reply]
Yes, I see it now. That is great! After the initial download, I will create the local config file and apply the changes.  Thanks
